gcc_11/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM ubuntu:focal
+
+LABEL maintainer="Luis Martinez de Bartolome <luism@jfrog.com>"
+
+ENV GCC_VERSION=11 \
+    CMAKE_VERSION_FULL=3.18.2 \
+    CC=/usr/bin/gcc \
+    CXX=/usr/bin/g++ \
+    PYENV_ROOT=/opt/pyenv \
+    PYTHON_VERSION=3.7.5 \
+    PATH=/opt/pyenv/shims:${PATH} \
+    DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get -qq update \
+    && apt-get -qq install -y --no-install-recommends \
+       sudo \
+       binutils \
+       wget \
+       git \
+       libc6-dev \
+       g++-${GCC_VERSION} \
```

Comment:
This image was not build as you have not added it to the `docker-compose.yml` file.

`gcc-11` does not exist for Ubuntu Focal